src/views/product-downloads-view/index.tsx

```diff
@@ -93,11 +93,20 @@ const ProductDownloadsViewContent = ({
 
 	// Group the selected release downloads by OS, for use in multiple places
 	const selectedRelease = releases.versions[currentVersion]
-	const downloadsByOS = useMemo(
+	let downloadsByOS = useMemo(
 		() => sortPlatforms(selectedRelease),
 		[selectedRelease]
 	)
 
+	if (currentProduct.slug === 'vagrant') {
+		downloadsByOS = {
+			...downloadsByOS,
+			linux: {
+				amd64: downloadsByOS.linux.amd64,
```

Comment:
this logic needs to have error handling. I selected an earlier version, and it caused an error
<img width="1149" alt="Screenshot 2024-11-05 at 12 05 28" src="https://github.com/user-attachments/assets/21d0a6bc-93ab-416f-87d2-7236b52d583b">

Reply:
I wasn't able to replicate this locally or in the [deploy preview](https://dev-portal-git-vagrant-update-install-binaries-wget-hashicorp.vercel.app/vagrant/install)... but added a snippet that should address it

```
	if (currentProduct.slug === 'vagrant') {
		downloadsByOS = {
			...downloadsByOS,
			linux: {
				amd64: downloadsByOS?.linux?.amd64 || null,
			},
		}
	}
```

Reply:
Unfortunately changing it `null` created another bug
<img width="1728" alt="Screenshot 2024-11-05 at 13 40 41" src="https://github.com/user-attachments/assets/5400e962-4298-48aa-a3b0-c678b19d9b0c">

I added a comment with code that will work here https://github.com/hashicorp/dev-portal/pull/2620#discussion_r1829843021

If you change the version to `1.4.0` or any version other than the default, you should get the error. I'm using FF